synok: limit number of jobs that can be scheduled at one time

ARTER616 opened the pull request:
fix(synok): set successfulJobsHistoryLimit and failedJobsHistoryLimit to 0, so completed and failed jobs do not store and concurrencyPolicy to Forbid, so new job will not start if existing one keeps running.

Closes #46

ARTER616 committed:
synok: limit number of jobs that can be scheduled at one time

fix(synok): set successfulJobsHistoryLimit and failedJobsHistoryLimit to 0, so completed and failed jobs do not store and concurrencyPolicy to Forbid, so new job will not start if existing one keeps running.

Closes #46

mmskv reviewed (changes requested):
Change successfulJobsHistoryLimit and failedJobsHistoryLimit to 3 for debugging purposes. When the job fails we want to have a way to check the reason why it failed.

ARTER616 committed:
synok: limit number of jobs that can be scheduled at one time

fix(synok): set successfulJobsHistoryLimit and failedJobsHistoryLimit to 3, so completed and failed jobs do not store more than 3 times and concurrencyPolicy to Forbid, so new job will not start if existing one keeps running.

Closes #46
